src/ninshiki_cpp/config/grpc/call_data_base.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2024 Ichiro ITS
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+// THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#include "ninshiki_cpp/config/grpc/call_data_base.hpp"
+
+namespace ninshiki_cpp
+{
+CallDataBase::CallDataBase() {}
```

Comment:
Just remove the constructor if it does not do anything.

Reply:
after I deleted the constructor or deleted the `call_data_base.cpp` file, when building the package it produced an error
```
/usr/bin/ld: libninshiki_cpp.so: undefined reference to `ninshiki_cpp::CallDataBase::CallDataBase()'
```

Reply:
You also need to remove the constructor from the class declaration in the `.hpp` file.